portal/src/views/Docs/Docs.vue

```diff
@@ -121,47 +178,183 @@ interface RouteDef {
 	description: string;
 	method: string;
 	path: string;
-	params: swag.Operation;
+	params: swag.Spec; // changed to swag.Spec because it fit our needs better than swag.Operation
 }
+
+// Copy to clipboard stuff - lets users copy the request url to their clipboard with ease
+const { isSupported, copy } = useClipboard();
 </script>
 
 <style scoped lang="scss">
+// TODO clean up these style defs in the future
+
 @import "@style/themes.scss";
 
+body {
+	margin: 0;
+	max-width: 100vw;
+}
+
+dl {
+	margin: 0 0 1em;
+	dt {
+		background-color: #ccc;
+		padding: 1em;
+		font-weight: bold;
+	}
+
+	dd {
+		padding: 0;
+		margin: 0;
+		border: 1px solid #ccc;
+		border-top: 0;
+		padding: 1em;
+	}
+}
+
+#app {
+	width: 100vw;
+	min-height: 100vh;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	flex-direction: column;
+	padding: 16px;
+}
+
+.accordions {
+	width: 100%;
+	max-width: 500px;
+}
+
+pre {
+	overflow-x: auto;
+	/* Use horizontal scroller if needed */
+	white-space: pre-wrap;
+	/* css-3 */
+	white-space: -moz-pre-wrap !important;
```

Comment:
Why did we use !important here?

Reply:
Also, with this CSS does the preprocessor not have styles for all browsers?

Reply:
i took that from a stack-overflow answer to fix the <pre> overflowing off of the screen, but not sure why they put that extra stuff in, maybe best practice? i took it out